src/getDataTransferItems.js

```diff
@@ -1,4 +1,5 @@
-export default function getDataTransferFiles(event, isMultipleAllowed = true) {
+/* @flow */
+export default function getDataTransferFiles(event: Object, isMultipleAllowed: boolean = true) {
```

Comment:
What exactly is an `event` ? apparently you want to express an object that has a `event.dataTransfer` attribute and a `event.target` attribute... why not make this explicit?